<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #281E5D;
QPushButton {
    background-color: #1520A6;
    color: white;
    font-family: Montserrat;
    font-weight: 600;
    font-size: 14px;
    text-align: center;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background-color: #1520A6;
    color: white;
    font-family: Montserrat;
    font-weight: 600;
    font-size: 18px;
    text-align: center;
}

QPushButton:pressed {
    background-color: #3944BB;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0,0">
           <item>
            <widget class="QPushButton" name="datos">
             <property name="text">
              <string>Datos paciente</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cargarimagen">
             <property name="text">
              <string>Cargar imagen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="imagenoriginal">
             <property name="text">
              <string>Imagen original</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="caracterizar">
             <property name="text">
              <string>Caracterizar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="diagnosticar">
             <property name="text">
              <string>Diagn√≥stico</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="guardar">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="datos_2">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="styleSheet">
                <string notr="true">font-family: Montserrat;
font-size: 20px;
font-weight: 600;
color: #FFFFFF;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="n1">
                  <property name="font">
                   <font>
                    <family>Montserrat</family>
                    <pointsize>-1</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Nombre y apellido</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="n2">
                  <property name="font">
                   <font>
                    <family>Montserrat</family>
                    <pointsize>-1</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel {
    font-family: Montserrat;
    font-size: 11px;
    font-weight: 600;
    qproperty-alignment: AlignCenter;
    color: #FFFFFF;

}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="d1">
                  <property name="font">
                   <font>
                    <family>Montserrat</family>
                    <pointsize>-1</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Documento de identidad</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="d2">
                  <property name="font">
                   <font>
                    <family>Montserrat</family>
                    <pointsize>-1</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel {
    font-family: Montserrat;
    font-size: 11px;
    font-weight: 600;
    qproperty-alignment: AlignCenter;
    color: #FFFFFF;

}</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <family>Montserrat</family>
                    <pointsize>-1</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Fecha de nacimiento</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDateEdit" name="dateEdit">
                  <property name="palette">
                   <palette>
                    <active>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>40</red>
                        <green>30</green>
                        <blue>93</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="ButtonText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>40</red>
                        <green>30</green>
                        <blue>93</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>40</red>
                        <green>30</green>
                        <blue>93</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="PlaceholderText">
                      <brush brushstyle="NoBrush">
                       <color alpha="128">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </active>
                    <inactive>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>40</red>
                        <green>30</green>
                        <blue>93</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="ButtonText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>40</red>
                        <green>30</green>
                        <blue>93</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>40</red>
                        <green>30</green>
                        <blue>93</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="PlaceholderText">
                      <brush brushstyle="NoBrush">
                       <color alpha="128">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </inactive>
                    <disabled>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>40</red>
                        <green>30</green>
                        <blue>93</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="ButtonText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>40</red>
                        <green>30</green>
                        <blue>93</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>40</red>
                        <green>30</green>
                        <blue>93</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="PlaceholderText">
                      <brush brushstyle="NoBrush">
                       <color alpha="128">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </disabled>
                   </palette>
                  </property>
                  <property name="font">
                   <font>
                    <family>Montserrat</family>
                    <pointsize>-1</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel {
    font-family: Montserrat;
    font-size: 11px;
    font-weight: 600;
    qproperty-alignment: AlignCenter;
    color: #FFFFFF;

}</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Sexo</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QComboBox" name="comboBox">
                  <property name="font">
                   <font>
                    <family>Montserrat</family>
                    <pointsize>-1</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Diagn√≥stico</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="diagline">
                  <property name="font">
                   <font>
                    <family>Montserrat</family>
                    <pointsize>-1</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel {
    font-family: Montserrat;
    font-size: 11px;
    font-weight: 600;
    qproperty-alignment: AlignCenter;
    color: #FFFFFF;

}</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Observaciones</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QTextEdit" name="obs">
                  <property name="font">
                   <font>
                    <family>Montserrat</family>
                    <pointsize>-1</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLineEdit {
    font-family: Montserrat;
    font-size: 11px;
    font-weight: 600;
    qproperty-alignment: AlignCenter;
    color: #FFFFFF;

}</string>
                  </property>
                  <property name="html">
                   <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:20px; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="im">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGraphicsView" name="imagenes"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="graf">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QGraphicsView" name="graficos"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Diag">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="diagnostico">
               <property name="styleSheet">
                <string notr="true">QLabel {
    font-family: Montserrat;
    font-size: 22px;
    font-weight: 400;
    qproperty-alignment: AlignCenter;
    color: #FFFFFF;

}</string>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGraphicsView" name="grafDiag"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
